weld/llvm.rs

```diff
@@ -387,16 +421,60 @@ impl LlvmGenerator {
                             .add(format!("{} = add i64 {}, {}", final_idx, start_str, offset));
                         final_idx
                     } else {
-                        idx_tmp.clone()
+                        if iter.kind == IterKind::FringeIter {
+                            let vector_len = format!("{}", vec_size(&elem_ty)?);
+                            let tmp = ctx.var_ids.next();
+                            let arr_len = ctx.var_ids.next();
+                            let offset = ctx.var_ids.next();
+                            let final_idx = ctx.var_ids.next();
+
+                            ctx.code
+                                .add(format!("{} = call i64 {}.size({} {})",
+                                arr_len,
+                                data_prefix,
+                                &data_ty_str,
+                                data_str));
```

Comment:
These are all because of `rustfmt`...should I change them? We've stopped using that tool because it changes how it formats code every minor release.